.github/test-spec.yml

```diff
@@ -31,7 +31,17 @@
   - "samples/subsys/mgmt/mcumgr/smp_svr/**/*"
 
 "CI-tfm-test":
-  - "**/*"
+  - "boards/arm/nrf5340dk_nrf5340/**/*"
+  - "boards/arm/nrf9160dk_nrf9160/**/*"
+  - "drivers/entropy/**/*"
+  - "drivers/serial/**/*"
+  - "drivers/timer/**/*"
```

Comment:
I don't understand why serial and timer is included.
If the logic is that TF-M has a serial and timer drivers, then it is TF-M specific drivers and not the zephyr ones, so that should not be a trigger.

Reply:
It might not be needed, I just saw that crypto tests compile source code in those directories so I assumed that tf-m tests do the same. I have perhaps been a bit too strict on the filter values, but I want to catch issues as early as possible. Especially  with serial since we use it extensively in our test system